classes/autotools.oeclass

```diff
@@ -48,7 +48,13 @@ autotools_configure[emit] = "do_configure"
 do_configure () {
     do_configure_autotools
 }
+AUTOCONFDIR = ""
+UPDATE_GNUCONF = ""
+UPDATE_GNUCONF:TARGET_CPU_aarch64 = "rm -f ${S}${AUTOCONFDIR}/config.{guess,sub}; cp ${TOPDIR}/meta/core/scripts/config.{guess,sub} ${S}${AUTOCONFDIR}"
```

Comment:
Please move this "one-liner" into do_configure_autotools, and use UPDATE_GNUCONF as a 0/1 flag instead.  This makes it easier to update config.{guess,sub} in other situations.